src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -36,17 +36,18 @@ const propTypes = {
 
     /** Holds data related to IOU view state, rather than the underlying IOU data. */
     iou: PropTypes.shape({
-
-        /** Whether or not the IOU step is loading (retrieving users preferred currency) */
-        loading: PropTypes.bool,
-
         /** Selected Currency Code of the current IOU */
         selectedCurrencyCode: PropTypes.string,
-    }).isRequired,
+    }),
 
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    iou: {
+        selectedCurrencyCode: null,
```

Comment:
We can have the `CONST.CURRENCY.USD` for the default value and clean up in here https://github.com/Expensify/App/blob/63d5a7e687fd30db85708503819e2ef7585c7994/src/pages/iou/steps/IOUAmountPage.js#L263